src/firebase/providers.js

```diff
@@ -0,0 +1,45 @@
+import { createUserWithEmailAndPassword, signInWithEmailAndPassword, getAuth } from 'firebase/auth'
+import { app } from '.'
+
+const FirebaseAuth = getAuth(app) 
```

Comment:
This is a variable that we may need in some other places in the future,

https://firebase.google.com/docs/auth/web/start#add-initialize-sdk

Take a look at the docs and see how they created a reference to the auth service, we can export the reference and get it from the firebase/index.js file